### PR TITLE
106 country name match

### PR DESCRIPTION
**Pull request by qifeng-bai**
Added some missing countries to current ALA file. Added codes to deal with country code

### CODE REVIEW

File: livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java [hunk 1]
```diff
@@ -364,6 +365,24 @@ public void assertCountryCentre() {
...
+   */
+  @Test
+  public void countryNameMatch() throws IOException {
+    Set<String> countryInFile = countryCentrePoints.keys();
+
+    for (Country country : Country.values()) {
+      String name = country.getTitle().toUpperCase();
+      String countryCode = country.getIso2LetterCode().toUpperCase();
+      if (!countryInFile.contains(name)) {
+        System.out.println(countryCode + " : " + country.getTitle());
```
Comment (djtfmartin) [inline livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java:381]: can we replace with log.info or log.debug ?

File: livingatlas/pipelines/src/test/java/au/org/ala/parser/LocationResourceParserTest.java [hunk 2]
```diff
@@ -26,15 +26,21 @@ public void setup() {
 
   @Test
   public void countryCentreTest() throws FileNotFoundException {
+    //    boolean result =
```
Comment (djtfmartin) [inline livingatlas/pipelines/src/test/java/au/org/ala/parser/LocationResourceParserTest.java:29]: Suggest we remove this commented out code

**Review by djtfmartin — CHANGES_REQUESTED** [review]
thanks @qifeng-bai  - just minor changes suggested

File: livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java (hunk 1 above)
Comment (qifeng-bai) [reply livingatlas/pipelines/src/test/java/au/org/ala/pipelines/interpreters/AlaLocationInterpreterTest.java:381]: Done

**Pull request merged**